futurex_openedx_extensions/helpers/models.py

```diff
@@ -306,10 +306,11 @@ class DataExportTask(models.Model):
 
     filename = models.CharField(max_length=255)
     view_name = models.CharField(max_length=255)
+    related_id = models.CharField(max_length=255, null=True, blank=True)
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     status = models.CharField(max_length=16, choices=STATUS_CHOICES, default=STATUS_IN_QUEUE)
     progress = models.FloatField(default=0.0)
-    notes = models.CharField(max_length=255, default='')
+    notes = models.CharField(max_length=255, default='', blank=True)
     tenant_id = models.IntegerField()
```

Comment:
This should be a foreign key.

Reply:
We started with foreign key, then there were issues in tests so for simplicity we used integer field. @shadinaif please add your comments here also was it a temporary change and should I work on it now ?

Reply:
fixed by @tehreem-sadat , with many thanks 🌷